tests/kubernetes-agent/KubernetesAgent.IntegrationTests/Setup/Common/TestLogger.cs

```diff
@@ -0,0 +1,228 @@
+using System;
+using System.Collections.Concurrent;
+using Halibut.Diagnostics;
+using Microsoft.VisualStudio.TestPlatform.ObjectModel;
```

Comment:
Some of these usings aren't needed, maybe run a "remove usings" over the solution

Reply:
Done!